tests/bsim/bluetooth/samples/battery_service/prj.conf

```
@@ -0,0 +1,14 @@
+CONFIG_BT=y
+CONFIG_LOG=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_PRIVACY=y
+CONFIG_BT_SMP=y
```

```suggestion
CONFIG_BT_PERIPHERAL=y
CONFIG_BT_SMP=y
```

Don't think we need privacy to test this feature, do we?